package.json

```diff
@@ -42,7 +42,11 @@
   },
   "devDependencies": {
     "rollup": "^2.58.0",
```

Comment:
rollup-plugin-dts@5.3.0 requires rollup@"^3.0.0", this should probably be updated to match?

Reply:
It works with a current version of rollup, I see no issues here